templates/lx_userdata.sh

```diff
@@ -237,6 +237,14 @@ install-docker() {
   apt-get -y install docker-ce docker-ce-cli containerd.io
 }
 
+# shellcheck disable=SC2317
+clone-watchmaker() {
+  if [[ -d "watchmaker" ]] ; then
+    try_cmd 1 rm -rf watchmaker
+  fi
```

Comment:
Probably don't even need the test honestly. Just remove it with prejudice!

```suggestion
rm -rf "$GIT_REPO"
```

Reply:
Done!